src/components/PeoplePage.tsx

```diff
@@ -1,35 +1,67 @@
+import { useEffect, useState } from 'react';
+
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
+import { Person } from '../types';
+import { getPeople } from '../api';
 import { PeopleTable } from './PeopleTable';
 
 export const PeoplePage = () => {
-  return (
-    <>
-      <h1 className="title">People Page</h1>
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [hasError, setHasError] = useState(false);
 
-      <div className="block">
-        <div className="columns is-desktop is-flex-direction-row-reverse">
-          <div className="column is-7-tablet is-narrow-desktop">
-            <PeopleFilters />
-          </div>
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        setHasError(false);
 
-          <div className="column">
-            <div className="box table-container">
-              <Loader />
+        const peopleFromServer = await getPeople();
 
-              <p data-cy="peopleLoadingError">Something went wrong</p>
+        setPeople(peopleFromServer);
+      } catch (err) {
+        setHasError(true);
+      } finally {
+        setIsLoading(false);
+      }
+    };
 
-              <p data-cy="noPeopleMessage">
-                There are no people on the server
-              </p>
+    fetchData();
+  }, []);
 
-              <p>There are no people matching the current search criteria</p>
+  return (
+    <>
```

Comment:
Would be better if you put some wrap tag in your components
```suggestion
    <div>
```